src/common/recordbatch/src/lib.rs

```diff
@@ -181,6 +192,26 @@ impl Stream for SimpleRecordBatchStream {
     }
 }
 
+/// Adapt a [Stream] of [RecordBatch] to a [RecordBatchStream].
+pub struct RecordBatchStreamAdaptor {
```

Comment:
Looks like `Adaptor` and `Adapter` are both used.

Reply:
Yes. I filed https://github.com/GreptimeTeam/greptimedb/issues/1665 to track